### DISCUSSION TITLE
TableColumn.Editable=false时列在表格中不可见

### DISCUSSION THREAD

**Discussion by SteelTeng** [dotnetcore/BootstrapBlazor#3322] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

8.4.1版本开始
Table组件中，把TableColumn.Editable设置为false会造成该列在表格中不显示

### Expected Behavior

在8.4.0及之前版本中，TableColumn.Editable只会影响新增和编辑对话框中的列可见性，不会影响表格中的列可见性
不知是否修改了Editable的作用范围，如果没有则希望修复此Bug。

### Steps To Reproduce

https://github.com/[SteelTeng/bb-demo-github

TableDemo.razor中
`<TableColumn @bind-Field="@context.Name" Text="姓名-Editable=true" Editable="true" />

<TableColumn @bind-Field="@context.Address" Text="地址-Editable=false" Editable="false" />`
正常情况下该2列都应该在表格中显示，但是实际上地址列并不会显示

### .NET Version

8.0

**Comment by bb-auto** [comment]
@SteelTeng Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment, marked as answer]
@SteelTeng 

https://github.com/dotnetcore/BootstrapBlazor/discussions/3249?wt.mc_id=DT-MVP-5004174
